sdk/python/pulumi_snowflake/failover_group.py

```diff
@@ -423,7 +423,7 @@ def __init__(__self__,
         target_failover_group = snowflake.FailoverGroup("targetFailoverGroup", from_replica=snowflake.FailoverGroupFromReplicaArgs(
             organization_name="...",
             source_account_name="...",
-            name=snowflake_failover_group["fg"]["name"],
+            name=%!v(PANIC=Format method: runtime error: invalid memory address or nil pointer dereference),
```

Comment:
This is a regression tracked in https://github.com/pulumi/pulumi/issues/12998 - we should deploy the P1 fix anyway and adopt a fix for the example generation later as it's available in pulumi/pulumi.